README.md

```diff
@@ -79,15 +79,12 @@ make run
 
 Alternatively, you could also run the application with
 [podman-compose](https://github.com/containers/podman-compose) by setting the
-`CACHITO_COMPOSE_ENGINE` variable (for now SELinux must be set to the
-**Permissive** mode before running the make command):
-
-> :warning: **Disabling SELinux or running it in Permissive mode may be
-> dangerous. Do it at your own risk and make sure you re-enable it after
-> running your integration tests.**
+`CACHITO_COMPOSE_ENGINE` variable to the path of the `podman-compose` script.
+Unfortunately, the latest release of `podman-compose` contains various bugs making
+it unusable for running Cachito locally. Use the script from the `devel` branch instead:
 
 ```bash
-make run CACHITO_COMPOSE_ENGINE=podman-compose
+make run CACHITO_COMPOSE_ENGINE=<path to podman-compose.py script>
```

Comment:
Should this make file include an option to fetch this podman compose script in a specific (compatible) version and setting this path here?

So there would be no doubts about what podman compose we want to run. It could also go into .gitignore so it could live within the project root.

Reply:
Oh, that's an interesting suggestion! Do you think it would be too "magical" if the Makefile does this?

Or do you mean, just provide the `git clone/checkout` commands here? I know it's described in the paragraph above, but sometimes people have the habit of skipping to the code blocks without reading the text first :)

Reply:
do you think it woul be too invasive if we did the magic? i.e., downloading the exact compatible version to that specific path and exporting the env variable.

Then newcomers would be able to run it without much trouble.

Reply:
What if we had a special value for `CACHITO_COMPOSE_ENGINE`, say `podman-compose-auto`. If the environment variable is set to that value, then we do the magic? Otherwise, we just use the path as is.

Reply:
Some questions:
Will you pull the scripts down at a specific commit or the latest on the `dev` branch? Do you want the default behavior to be "in the business" of maintaining what the latest non-broken scripts are? Would it be easier to make it a submodule in this repo (that would not be as clean though)?

I think doing the magic would be good for those that want to start using podman but might not want to go through the effort of troubleshooting/debugging.

Reply:
I was thinking about doing it for a specific revision. I also think this would be a temporary measure until `podman-compose` compatible releases hit the stable versions of major distros (most of us work with Fedora). 

@lcarva I like the `podman-compose-auto` idea. If that is too much trouble, this looks good already and could be done in a future PR.

Reply:
Alright, I added the `podman-compose-auto` alongside with an update to the README.md file.